analyzer/src/main/kotlin/managers/Cargo.kt

```diff
@@ -138,8 +138,9 @@ class Cargo(
     private fun resolveLockfile(metadata: JsonNode): File {
         val workspaceRoot = metadata["workspace_root"].textValueOrEmpty()
         val lockfile = File(workspaceRoot, "Cargo.lock")
-        if (!lockfile.isFile) {
-            throw IllegalArgumentException("missing Cargo.lock file")
+
+        require(lockfile.isFile) {
```

Comment:
out of scope: Should this adhere to allowDynamic version as well? e.g. be moved into the caller?

Reply:
I wanted to refactor this anyway after #1789 is merged, to align it with how it is implemented in the other package managers.

Reply:
Great, so I'll merge this as-is and leave the rest to you 😄

Reply:
The alignment PR is ready: #1794